### PR TITLE
Conrad updated the API, Frank updated the test

### PR DESCRIPTION
**Pull request by frank-24**
Conrad updated the API:
1. `T1w` and `bold` images are put into 2 separate end point
2. updated settings

Frank updated the test:
1. updated test according to the API changes

### CODE REVIEW

**Comment by chrisgorgo** [comment]
Does this supersede #12?

**Comment by 394822740** [comment]
Yes it supersedes #12

File: dockereve-master/eve-app/settings.py [hunk 1]
```diff
@@ -8,38 +8,63 @@
     'MONGO_DBNAME': 'scenarios',
     'X_DOMAINS': '*',
     'DOMAIN': {
-        'scenarios': {
-            'item_title': 'scenario',
+         'bold': {
+            'item_title': 'bold-2',
```
Comment (chrisgorgo) [inline dockereve-master/eve-app/settings.py]: Why 'bold-2'?

File: dockereve-master/eve-app/settings.py [hunk 2]
```diff
@@ -8,38 +8,63 @@
...
             'resource_methods': ['GET', 'POST'],
             'item_methods': ['GET', 'PATCH', 'PUT', 'DELETE'],
             'schema': {
-                # 'title': {
-                #     'type': 'string',
-                #     'minlength': 1,
-                #     'maxlength': 100,
-                #     'required': True
-                # },
```
Comment (chrisgorgo) [inline dockereve-master/eve-app/settings.py:21]: Why is this commented out? Should it be removed?

File: dockereve-master/eve-app/settings.py [hunk 3]
```diff
@@ -104,14 +204,109 @@
...
+                'CogAtlasID' : {'type': 'string'},
+                'CogPOID': {'type': 'string'},
+                'InstitutionName': {'type': 'string'},
+                'InstitutionAddress': {'type': 'string'},
+                'ConversionSoftware': {'type': 'string'},
+                'ConversionSoftwareVersion': {'type': 'string'},
+            } #end of schema
+        },#end of bold
+          'T1w': {
+            'item_title': 'T1w-',
```
Comment (chrisgorgo) [inline dockereve-master/eve-app/settings.py]: Why the dash at the end?

**Review by chrisgorgo — APPROVED** [review]
I feel that it would be more concise to set all of the `'required': True` in a programmatic way (in a loop), but this is a good start.

File: dockereve-master/eve-app/settings.py (hunk 3 above)
Comment (394822740) [reply dockereve-master/eve-app/settings.py]: Good point ; just cleaned up and pushed the updated version to my branch again. Should make another PR or wait until next time together.

**Comment by chrisgorgo** [comment]
Feel free to merge this whenever you want.

**Pull request merged**